packages/sil-voting-app/src/index.css

```diff
@@ -0,0 +1,11 @@
+body {
```

Comment:
Replace the main css using global styles, following the same approach as the globalStyles in bierzo-wallet.